src/_guides/language/effective-dart/design.md

```diff
@@ -1682,12 +1682,12 @@ DateTime(int year,
     int microsecond = 0]);
 
 Duration(
-    {int days: 0,
-    int hours: 0,
-    int minutes: 0,
-    int seconds: 0,
-    int milliseconds: 0,
-    int microseconds: 0});
+    {int days = 0,
+    int hours = 0,
+    int minutes = 0,
+    int seconds = 0,
+    int milliseconds = 0,
+    int microseconds = 0});
```

Comment:
Thanks for spotting this. Could you also update the source from which this excerpt is taken (that'll get the build to pass on Travis). It is in `misc/lib/effective_dart/design_good.dart`; search for the region named `omit-optional-positional`.